packages/features/bookings/Booker/components/hooks/useBookingForm.ts

```diff
@@ -27,7 +27,7 @@ export const useBookingForm = ({ event }: IUseBookingForm) => {
     .object({
       responses: event?.data
         ? getBookingResponsesSchema({
-            eventType: event?.data,
+            bookingFields: event.data.bookingFields,
```

Comment:
We just need bookingFields in here so there's no need to pass the whole eventType

Reply:
Great fix for Law of Demeter here 🙏🏼